doc/source/user_guide/tui_commands.rst

```diff
@@ -20,6 +20,19 @@ session easier, you can install a tool such as
 both command line completion and history. To inspect any PyFluent TUI object further,
 you can use the Python built-in `help <https://docs.python.org/3/library/functions.html#help>`_
 and `dir <https://docs.python.org/3/library/functions.html#dir>`_ functions.
+For example, to see the options available under the viscous model menu, the
+following can be used assuming that ``solver`` is the session instance returned
+by ``launch_fluent``:
+
+.. code:: python
+
+   dir(solver.tui.define.models.viscous)
```

Comment:
I think it would be better if we showed an example of what the output looks like (can truncate it if it is too long, or find another command with shorter output to use as example), for example:
```suggestion
.. code:: python

   >>> dir(solver.tui.define.models.viscous)
   ['detached_eddy_simulation', 'inviscid', 'k_kl_w', 'ke_realizable', 'ke_rng', 'ke_standard', 'kw_bsl', 'kw_geko', 'kw_sst', 'kw_standard', 'kw_wj_bsl_earsm', 'laminar', 'large_eddy_simulation', 'mixing_length', 'reynolds_stress_model', 'sas', 'spalart_allmaras', 'transition_sst']
```
This should be rendered nicely in the doc output

Same for `help()` below